tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -98,7 +98,7 @@ declare var cfg:ISnippetConfig;
             let ingest = conString[strIngestionendpoint];
             let endpointUrl = ingest ? ingest + "/v2/track" : aiConfig.endpointUrl; // only add /v2/track when from connectionstring
 
-            let message = "SDK LOAD Failure: Failed to load Application Insights SDK script (See stack for details)";
+            let message = "SDK LOAD Failure: Failed to load Application Insights SDK script via " + targetSrc + " (See stack for details)";
```

Comment:
I don't think we need to add the targetSrc here as it gets included in the message details via the `_createException` and `_createInternal`, so this becomes a duplication